protostar-rust/src/main.rs

```diff
@@ -66,16 +77,38 @@ fn dependencies_for_package(
 fn main_execution() -> Result<()> {
     let _args = Args::parse();
 
+    let project_root_path = project_root::get_project_root()?
+        .to_str()
+        .expect("error reading project root path")
+        .to_owned();
```

Comment:
We shoudn't rely on functions like this. The new runner will run all tests in scarb workspace, we have to resolve paths like this using scarb metadata.